storm-core/src/ui/public/index.html

```diff
@@ -106,92 +106,92 @@ <h2>Nimbus Configuration</h2>
     $.blockUI({ message: '<img src="images/spinner.gif" /> <h3>Loading summary...</h3>'});
 });
 $(document).ready(function() {
-    $.extend( $.fn.dataTable.defaults, {
-      stateSave: true,
-      lengthMenu: [[20,40,60,100,-1], [20, 40, 60, 100, "All"]],
-      pageLength: 20
-    });
-
-    $.ajaxSetup({
-        "error":function(jqXHR,textStatus,response) {
-            var errorJson = jQuery.parseJSON(jqXHR.responseText);
-            getStatic("/templates/json-error-template.html", function(template) {
-                $("#json-response-error").append(Mustache.render($(template).filter("#json-error-template").html(),errorJson));
-            });
-        }
-    });
-    var uiUser = $("#ui-user");
-    var clusterSummary = $("#cluster-summary");
-    var clusterResources = $("#cluster-resources");
-    var nimbusSummary = $("#nimbus-summary");
-    var ownerSummary = $("#owner-summary");
-    var topologySummary = $("#topology-summary");
-    var supervisorSummary = $("#supervisor-summary");
-    var config = $("#nimbus-configuration");
-
-    getStatic("/templates/index-page-template.html", function(indexTemplate) {
-        $.getJSON("/api/v1/cluster/summary",function(response,status,jqXHR) {
-            getStatic("/templates/user-template.html", function(template) {
-                uiUser.append(Mustache.render($(template).filter("#user-template").html(),response));
-                $('#ui-user [data-toggle="tooltip"]').tooltip()
-            });
-
-            clusterSummary.append(Mustache.render($(indexTemplate).filter("#cluster-summary-template").html(),response));
-            $('#cluster-summary [data-toggle="tooltip"]').tooltip();
-
-            clusterResources.append(Mustache.render($(indexTemplate).filter("#cluster-resources-template").html(),response));
-            $('#cluster-resources [data-toggle="tooltip"]').tooltip();
+    $.getJSON("/api/v1/cluster/configuration",function(response,status,jqXHR) {
```

Comment:
nit: Could we rename this configuration response and status?  They are over-ridden by the other ajax calls and I would prefer that it is not confusing in the code which variable is being accessed at which time.

Reply:
Done!